connect/runtime/src/main/java/org/apache/kafka/connect/runtime/SinkConnectorConfig.java

```diff
@@ -108,6 +132,20 @@ public static boolean hasTopicsRegexConfig(Map<String, String> props) {
         return topicsRegexStr != null && !topicsRegexStr.trim().isEmpty();
     }
 
+    public static boolean hasDlqTopicConfig(Map<String, String> props) {
+        String dqlTopicStr = props.get(DLQ_TOPIC_NAME_CONFIG);
+        return dqlTopicStr != null && !dqlTopicStr.trim().isEmpty();
+    }
+
+    public static List<String> parseTopicsList(Map<String, String> props) {
+        List<String> topics = (List<String>) ConfigDef.parseType(TOPICS_CONFIG, props.get(TOPICS_CONFIG), Type.LIST);
+        return topics
+                .stream()
+                .filter(topic -> !topic.isEmpty())
+                .distinct()
```

Comment:
Is this necessary? Doesn't seem like it actually breaks anything if the user specifies duplicate topics. And if we do want to enforce constraints about non-empty topics and/or no duplicated topics, we can do that with a `Validator` for that property when it's [defined in the ConfigDef](https://github.com/apache/kafka/blob/9bc96d54f8953d190a1fb6478a0656f049ee3b32/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/SinkConnectorConfig.java#L68).

Reply:
Actually it isn't. Just for the shake of having a "cleaned" list of topics. For instance, when logging topics in which the connector is going to subscribe...

Reply:
👍 fair enough

Reply:
why not just return a Set instead of List? 

```
return topics
                .stream()
                .filter(topic -> !topic.isEmpty())
                .collect(Collectors.toSet());
```

Reply:
That what I thought but we would also have a topic name with an empty string.